play-services-base/core/src/main/kotlin/org/microg/gms/settings/SettingsProvider.kt

```diff
@@ -117,6 +117,7 @@ class SettingsProvider : ContentProvider() {
             CheckIn.SECURITY_TOKEN -> checkInPrefs.getLong(key, 0)
             CheckIn.VERSION_INFO -> checkInPrefs.getString(key, "") ?: ""
             CheckIn.DEVICE_DATA_VERSION_INFO -> checkInPrefs.getString(key, "") ?: ""
+            CheckIn.HIDE_APP_ICON -> getSettingsBoolean(key, true)
```

Comment:
Here you specify that the default value is true.

Reply:
So what's the requested change here?

Reply:
Using `false` will make the default not hidden:
`CheckIn.HIDE_APP_ICON -> getSettingsBoolean(key, false)`

Reply:
Done.